src/pages/accountSetup/AccountSetupPage.jsx

```diff
@@ -0,0 +1,378 @@
+import {
+  Button,
+  CircularProgress,
+  FormControl,
+  FormHelperText,
+  styled,
+  Box,
+  Checkbox,
+} from "@mui/material";
+import * as yup from "yup";
+import { useFormik } from "formik";
+import FirestoreDocumentData from "../../hooks/user/useUserScopeDocument";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import Page from "../../components/page/Page";
+import { updateDoc } from "firebase/firestore";
+import CustomContainer from "../../components/container/CustomContainer";
+import CustomSelect from "../../components/customSelect/CustomSelect";
+import { Typography } from "@mui/material";
+import Grid from "@mui/material/Grid";
+import CustomTextField from "../../components/textField/CustomTextField";
+
+// Validation schema, update where necessary.
+const accountSetupSchema = yup.object({
+  headingsFont: yup.string().required("Please select a heading font"),
+  headingPointSize: yup
+    .number()
+    .required("Please select a heading point size")
+    .min(10, "Please select a heading point size from 10 to 30")
+    .max(25, "Please select a heading point size from 10 to 30"),
+  mainTextFont: yup.string().required("Please select a main text font"),
+  mainTextPointSize: yup
+    .number()
+    .required("Please select a main text point size")
+    .min(10, "Please select a main text point size from 10 to 30")
+    .max(25, "Please select a main text point size from 10 to 30"),
```

Comment:
This does not match with the scale. Should probably be 30 🤔 
```suggestion
    .max(30, "Please select a main text point size from 10 to 30"),
```